package.json

```diff
@@ -17,59 +17,49 @@
     "which": "^1.2.10"
   },
   "devDependencies": {
-    "babel-cli": "^6.16.0",
     "babel-eslint": "^7.1.1",
-    "babel-plugin-check-es2015-constants": "^6.8.0",
-    "babel-plugin-transform-es2015-arrow-functions": "^6.8.0",
-    "babel-plugin-transform-es2015-block-scoped-functions": "^6.8.0",
-    "babel-plugin-transform-es2015-block-scoping": "^6.18.0",
-    "babel-plugin-transform-es2015-classes": "^6.18.0",
-    "babel-plugin-transform-es2015-computed-properties": "^6.8.0",
-    "babel-plugin-transform-es2015-destructuring": "^6.18.0",
-    "babel-plugin-transform-es2015-duplicate-keys": "^6.8.0",
-    "babel-plugin-transform-es2015-for-of": "^6.18.0",
-    "babel-plugin-transform-es2015-function-name": "^6.9.0",
-    "babel-plugin-transform-es2015-literals": "^6.8.0",
-    "babel-plugin-transform-es2015-object-super": "^6.8.0",
-    "babel-plugin-transform-es2015-parameters": "^6.18.0",
-    "babel-plugin-transform-es2015-shorthand-properties": "^6.18.0",
-    "babel-plugin-transform-es2015-spread": "^6.8.0",
-    "babel-plugin-transform-es2015-sticky-regex": "^6.8.0",
-    "babel-plugin-transform-es2015-template-literals": "^6.8.0",
-    "babel-plugin-transform-es2015-unicode-regex": "^6.11.0",
-    "babel-plugin-transform-regenerator": "^6.16.1",
+    "babel-preset-es2015": "^6.24.0",
     "babelify": "^7.3.0",
     "browserify": "^13.0.1",
     "ethereum-common": "0.0.18",
+    "ethereumjs-block": "^1.2.2",
+    "ethereumjs-tx": "^1.1.1",
     "ethereumjs-util": "^4.5.0",
+    "ethereumjs-vm": "^2.0.1",
     "fast-async": "^6.1.2",
     "http-server": "^0.9.0",
     "nightwatch": "^0.9.5",
+    "selenium-standalone": "^6.0.1",
     "solc": "^0.4.3",
     "standard": "^7.0.1",
     "standard-reporter": "^1.0.5",
     "tape": "^4.6.0",
     "web3": "^0.15.3",
     "yo-yo": "^1.2.1",
-    "yo-yoify": "^3.1.0",
-    "ethereumjs-block": "^1.2.2",
-    "ethereumjs-tx": "^1.1.1",
-    "ethereumjs-vm": "^2.0.1"
+    "yo-yoify": "^3.1.0"
   },
   "scripts": {
-    "start_node": "./runNode.sh",
-    "start_eth": "npm run warning_message; eth -j --rpccorsdomain '*'",
-    "start_geth": "npm run warning_message; geth --rpc --rpcapi 'web3,eth,debug' --rpcport 8545 --rpccorsdomain '*'",
-    "build": "mkdir build; browserify src/index.js -g yo-yoify -o build/app.js -t [ babelify ]; babel --plugins=transform-es2015-template-literals build/app.js --out-file build/app.js",
-    "test": "standard; babel src --out-dir babelify-src; tape ./test/tests.js",
-    "serve": "http-server .",
+    "build": "mkdirp build; browserify src/index.js > build/app.js",
+    "lint": "standard | notify-error",
     "nightwatch_local": "nightwatch --config nightwatch.js --env local",
-    "nightwatch_remote_firefox": "nightwatch --config nightwatch.js --env default",
     "nightwatch_remote_chrome": "nightwatch --config nightwatch.js --env chrome",
-    "nightwatch_remote_safari": "nightwatch --config nightwatch.js --env safari",
```

Comment:
oh no, safari is still in there. it's just now all alphabetically sorted. The safari test is not removed.